# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/focus/support/activeelement-after-immediately-focusing-same-site-iframe-inner-contentwindow.html
# Commit: c63ecdad6a3c
# Full Hash: c63ecdad6a3c60ba625517d34bb29c8aae9f5e95
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-02-22 17:34:52
# Regressor Bug: 1663931
# File Overlap Count: 1
# Description:
#   Bug 1663931 - Avoid moving focus when changing iframe remoteness. r=nika,mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99215
# ==============================================================================

diff -r 828c62bfc0a2 -r c63ecdad6a3c testing/web-platform/tests/focus/support/activeelement-after-immediately-focusing-same-site-iframe-inner-contentwindow.html
--- a/testing/web-platform/tests/focus/support/activeelement-after-immediately-focusing-same-site-iframe-inner-contentwindow.html	Mon Feb 22 10:48:53 2021 +0000
+++ b/testing/web-platform/tests/focus/support/activeelement-after-immediately-focusing-same-site-iframe-inner-contentwindow.html	Mon Feb 22 10:51:51 2021 +0000
@@ -8,7 +8,9 @@
 <h1>Inner</h1>
 <script>
 document.body.onfocus = function() {
-    parent.postMessage("innerbodyfocus,", "*");
+    // Commented out pending resolution of
+    // https://github.com/whatwg/html/issues/6209
+    // parent.postMessage("innerbodyfocus,", "*");
 }
 document.body.onblur = function() {
     parent.postMessage("innerbodyblur,", "*");